drivers/exec/driver.go

```diff
@@ -159,6 +166,9 @@ type Config struct {
 	// AllowCaps configures which Linux Capabilities are enabled for tasks
 	// running on this node.
 	AllowCaps []string `codec:"allow_caps"`
+
+	DeniedHostUidsStr string `codec:"denied_host_uids"`
```

Comment:
A total nit, but it looks like these variables were suffixed with `Str` because there used to be other variables of the same name that have since been removed.  Would it make sense to remove the suffix now?

Reply:
you are right, I picked this one from someone else :)